src/jestGlobalMocks.ts

```diff
@@ -0,0 +1,15 @@
+const mock = () => {
+  let storage = {};
+  return {
+    getItem: key => key in storage ? storage[key] : null,
+    setItem: (key, value) => storage[key] = value || '',
+    removeItem: key => delete storage[key],
+    clear: () => storage = {},
+  };
+};
+
+Object.defineProperty(window, 'localStorage', { value: mock() });
+Object.defineProperty(window, 'sessionStorage', { value: mock() });
+Object.defineProperty(window, 'getComputedStyle', {
+  value: () => ['-webkit-appearance']
+});
```

Comment:
Is this just a placeholder for future mocks? I was just wondering about the use of window storage?

Reply:
This is good - we can use Mocha for browser testing, but there might not be any point in this project (we'll use something more app-y for our end to end testing)